app/build.gradle

```diff
@@ -252,5 +252,7 @@ lokaliseProperties.load(new FileInputStream(rootProject.file("lokalise.propertie
 lokalise {
     id lokaliseProperties['id']
     token lokaliseProperties['token']
+
+    downloadConfig {}
 }
```

Comment:
Can you explain what this part do?

Reply:
Yes! It creates a `DownloadConfig`-object for configuration purposes, but does not change any of the default values.

In Kotlin-terms it would be like doing the following:

```kotlin
data class DownloadConfig(
    val exampleConfigProp: Boolean = true
)

val downloadConfig = DownloadConfig()
```